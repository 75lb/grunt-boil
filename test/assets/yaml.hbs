---
word: CLIVE
related: 
  - bread
  - milk
---
<html>
<body>
<h1>{{word}}</h1>
<ul>
  {{~#each related}}
  <li>{{this}}</li>
  {{~/each}}
</ul>
</body>
</html>
