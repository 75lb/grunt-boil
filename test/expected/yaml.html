<html>
<body>
<h1>CLIVE</h1>
<ul>
  <li>bread</li>
  <li>milk</li>
</ul>
</body>
</html>